client/changelist/change.go

```diff
@@ -20,7 +20,7 @@ const (
 // all changes in Snapshot and Timestamp are programmatically
 // generated base on Root and Targets changes.
 const (
-	TypeRootRole          = "role"
+	TypeRootRole          = "root"
```

Comment:
I agree that this is confusing naming, but if you look where this constant is used it looks like this is referring to "root role" as in "not-delegated" e.g. `root`, `snapshot`, `target`, and `timestamp`.

E.g. see https://github.com/docker/notary/blob/b4d4ab6c96e19c9f679f59270130966eb42fbec9/client/helpers.go#L176 , where it refers to any "base" role change. 

I think it makes more sense to change the constant to `TypeRole` or maybe `TypeBaseRole`